jellyfin_kodi/setup.py

```diff
@@ -18,50 +18,10 @@ class Setup(object):
 
     def __init__(self):
 
-        self.set_web_server()
         self.setup()
 
         LOG.info("---<[ setup ]")
 
-    def set_web_server(self):
-
-        ''' Enable the webserver if not enabled. This is used to cache artwork.
-            Will only test once, if it fails, user will be notified only once.
-        '''
-        if settings('enableTextureCache.bool'):
-
-            get_setting = JSONRPC('Settings.GetSettingValue')
-
-            if not self.get_web_server():
-
-                set_setting = JSONRPC('Settings.SetSetingValue')
-                set_setting.execute({'setting': "services.webserverport", 'value': 8080})
-                set_setting.execute({'setting': "services.webserver", 'value': True})
-
-                if not self.get_web_server():
-
-                    settings('enableTextureCache.bool', False)
-                    dialog("ok", heading="{jellyfin}", line1=translate(33103))
```

Comment:
This probably needs to get removed from the strings files as well.  Not sure if it's best to do that here or through weblate.